eng/pipelines/mgmt.yml

```diff
@@ -202,9 +202,7 @@ stages:
           Scope: ${{ parameters.Scope }}
           ShouldPublish: ${{ parameters.ShouldPublish }}
 
-  # The Release job is conditioned on whether we are building a pull request and the branch.
-  - ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal')) }}:
-    - template: /eng/pipelines/templates/jobs/mgmt-release.yml
-      parameters:
-        ShouldPublish: ${{ parameters.ShouldPublish }}
-        DependsOn: Build
\ No newline at end of file
+      - ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), eq(parameters.ShouldPublish, 'true')) }}:
```

Comment:
I don't think this will be consume in the public but just in case I think it is worth keeping the internal check.